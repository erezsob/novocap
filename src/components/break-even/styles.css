.container {
    display: flex;
    flex-direction: row;
    justify-content: space-between;
    align-items: center;
    height: 20px;
    width: 90%;
    padding-top: 20px;
    color: white;
    font-weight: bold;
    font-size: 20px;
}

.barContainer {
    margin: 20px;
    width: 150px;
    height: 150px;
    position: relative;
    display: flex;
    flex-direction: column;
    justify-content: center;
}

.title {
    display: flex;
    flex-direction: column;
    padding-top: 13px;
}

.date {
    font-size: 11px;
    font-weight: 300;
}

.addCosts {
    color: #fff;
    font-size: 19px;
}