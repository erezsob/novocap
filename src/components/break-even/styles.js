import { blueMain } from '../../common/variables.css';

export default {
  card: {
    display: 'flex',
    flexDirection: 'column',
    alignItems: 'center',
    boxShadow: 'none',
    height: 208,
    backgroundColor: blueMain,
    cursor: 'pointer'
  },
  iconButton: {
    color: 'white',
    width: 24
  }
};
