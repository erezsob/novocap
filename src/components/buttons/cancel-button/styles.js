import { pinkyRed } from '../../../common/variables.css';

export default {
  cancel: {
    color: pinkyRed
  }
};
