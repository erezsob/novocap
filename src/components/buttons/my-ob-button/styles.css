@value blueLight, bpXSmall from '../../../common/variables.css';

.button {
    border-radius: 4px;
    background-color: blueLight;
    color: white;
    padding: 8px 0;
    font-size: 12px;
    display: table;
    border: 0;
    outline: 0;
    cursor: pointer;
}

.bigger {
    padding: 10px 0;
    font-size: 16px;
}

.icon,
.text {
    vertical-align: middle;
    text-align: center;
}

.text {
    padding: 0 20px;
    font-weight: lighter;
}

.icon ~ .text {
    display: none;
}

.icon {
    display: table-cell;
    padding: 0 10px;
    position: relative;
    top: 2px;
}

@media (min-width: bpXSmall) and (min-device-width: bpXSmall) {
    .icon ~ .text {
        padding-left: 0;
        display: table-cell;
    }
}
