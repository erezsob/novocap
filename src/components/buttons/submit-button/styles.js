import { greenGrass } from '../../../common/variables.css';

export default {
  submit: {
    color: greenGrass
  }
};
