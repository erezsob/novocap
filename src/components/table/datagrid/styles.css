@value defaultText, gray98, gainsboro from '../../../common/variables.css';

.ReactTable {
    border: inherit;
}

.ReactTable .rt-table {
    border-radius: 4px;
}

.ReactTable .rt-thead {
    background-color: white;
    color: defaultText;
    text-transform: uppercase;
    font-size: 10px;
}

.ReactTable .rt-tbody .rt-td,
.ReactTable .rt-thead .rt-th {
    display: flex;
    justify-content: center;
    align-items: center;
    white-space: normal;
    border-right: 1px solid gainsboro;
}

.ReactTable .rt-th{
    outline: none;
}

.ReactTable .rt-tbody .rt-tr-group {
    color: #323435;
    background-color: gray98;
    font-size: 13px;
    border-bottom: inherit;
}

.ReactTable .rt-thead,
.ReactTable .rt-tr-group {
    text-align: center;
    font-weight: bold;
    height: 60px;
}

.ReactTable .rt-tr.-even {
    background-color: white;
}

.ReactTable.-striped .rt-tr.-odd {
    background-color: gray98;
}
