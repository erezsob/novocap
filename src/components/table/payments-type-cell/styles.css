@value defaultText from '../../../common/variables.css';

.item {
    padding-right: 7px;
    padding-left: 7px;
    border-right: 1px solid defaultText;
}

.item:first-of-type {
    padding-left: 0;
}

.item:last-of-type {
    padding-right: 0;
    border-right: inherit;
}