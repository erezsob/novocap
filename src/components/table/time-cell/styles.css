@value defaultText, bpSmall from '../../../common/variables.css';

.dateHours,
.dateMonth {
    display: block;
}

.dateMonth {
    color: defaultText;
}

@media (min-width: bpSmall) and (min-device-width: bpSmall) {
    .dateHours,
    .dateMonth {
        display: inline-block;
    }

    .dateHours {
        padding-left: 7px;
    }

    .dateMonth {
        padding-right: 7px;
        border-right: 1px solid defaultText;
    }
}