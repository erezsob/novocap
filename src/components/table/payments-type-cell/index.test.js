import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import PaymentTypesCell from './';
import styles from './styles.css';

describe('<PaymentTypesCell />', () => {
  let wrapper;
  const paymentTypes = [
    {
      name: 'Cash'
    },
    {
      name: 'EC'
    }
  ];

  beforeEach(() => {
    wrapper = shallow(<PaymentTypesCell paymentTypes={paymentTypes} />);
  });

  it('renders', () => {
    expect(wrapper).toHaveLength(1);
  });

  it('renders properly', () => {
    const tree = renderer
      .create(<PaymentTypesCell paymentTypes={paymentTypes} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders with the right css class', () => {
    expect(wrapper.find(`.${styles.item}`)).toHaveLength(paymentTypes.length);
  });
});
