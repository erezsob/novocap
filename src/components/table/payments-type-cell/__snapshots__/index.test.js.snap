// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PaymentTypesCell /> renders properly 1`] = `
<span>
  <span
    className="item"
  >
    Cash
  </span>
  <span
    className="item"
  >
    EC
  </span>
</span>
`;
